### PR TITLE
potential fix for assertion error thrown due to MD5 missing from some…

### PR DESCRIPTION
**Pull request by marwanad**

### CODE REVIEW

**Comment by marwanad** [comment]
@gcgrant @michaelcosta

File: src/main/java/com/mixpanel/android/util/HttpService.java [hunk 1]
```diff
@@ -115,6 +115,9 @@ public boolean isOnline(Context context) {
                     throw e;
                 }
             }
+            catch (Throwable e) {
+                Log.e(LOGTAG, "Caught Throwable", e);
```
Comment (marwanad) [inline src/main/java/com/mixpanel/android/util/HttpService.java]: should we be doing something more meaningful here ?

Comment (michaelcosta) [inline src/main/java/com/mixpanel/android/util/HttpService.java]: It should maybe not be error level logging, and shoudl be wrapped in the `if (MPConfig.DEBUG) {`

Comment (michaelcosta) [inline src/main/java/com/mixpanel/android/util/HttpService.java]: maybe debug level

**Comment by michaelcosta** [comment]
LGTM

**Comment by michaelcosta** [comment]
We should update the version number so that we can put it up on our nexus repo as well

**Comment by marwanad** [comment]
This is now on our nexus repo

**Pull request merged**